tests/CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ foreach(test ${Specs})
     ADD_TEST(${test}   rspec --format doc ${test})
 endforeach(test)
 
-ADD_TEST("integration"  ruby ${CMAKE_CURRENT_SOURCE_DIR}/integration/run.rb)
+# disable integration test as it is too fragile on non intel architectures
+# ADD_TEST("integration"  ruby ${CMAKE_CURRENT_SOURCE_DIR}/integration/run.rb)
 ADD_TEST("translations" rspec --format doc ${CMAKE_CURRENT_SOURCE_DIR}/integration/translations_spec.rb)
 
 file(GLOB libyui_specs "libyui/*_spec.rb")
```

Comment:
BTW these tests also use tmux and libyui. Depending on what is fragile, we may want to disable these too.